### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nadheesh**
## Purpose
Fix readme.md

### CODE REVIEW

**Review by malithjayasinghe — DISMISSED** [review]

**Review by AathmanT — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -30,11 +29,11 @@ Execute the commands below to build from the source.
...
 * To build the package:
    ```   
-   bal build -c
+   bal build ./ballerina
    ```
 * To build the package without tests:
    ```
-   bal build -c --skip-tests
+   bal test ./ballerina
    ```
```
Comment (NipunaRanasinghe) [inline README.md:37]: `bal build` will build the package without running tests. So IMO we can have only one sub topic instead of having 2.

**Review by NipunaRanasinghe — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -56,4 +55,4 @@ All contributors are encouraged to read the [Ballerina Code of Conduct](https://
 
 * Discuss code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
 * Chat live with us via our [Slack channel](https://ballerina.io/community/slack/).
```
Comment (NipunaRanasinghe) [inline README.md:57]: Lets replace the slack channel link with the Ballerina discord server link

Comment (NipunaRanasinghe) [reply README.md:57]: `* Chat live with us via our [Discord server](https://discord.gg/ballerinalang).`

File: README.md (hunk 1 above)
Comment (nadheesh) [reply README.md]: I changed it accordingly following github module

File: README.md (hunk 2 above)
Comment (nadheesh) [reply README.md]: fixed

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Pull request merged**